xbmc/cores/VideoPlayer/VideoPlayer.h

```diff
@@ -323,6 +323,7 @@ class CVideoPlayer : public IPlayer, public CThread, public IVideoPlayer,
   int GetChapter() const override;
   void GetChapterName(std::string& strChapterName, int chapterIdx = -1) const override;
   int64_t GetChapterPos(int chapterIdx = -1) const override;
+  int GetPreviousChapter();
```

Comment:
Maybe move to line 372 after `void SetUpdateStreamDetails();` as all others are override and usually overrides are enumerated fist.

Reply:
I wasn't sure. makes sense though, I also made it protected, we tend to make too much stuff public for no reason.